### PR TITLE
bastion: move to mongodb6 + python2 -> 3 + set mongodb to use always use the same log file + build for arm64

### PR DESCRIPTION
**Pull request by QuintenQVD0**
## Description

- Move to mongoDB 6 as it is longer supported and recommended to use
- change the mongodb startup cmd to always use the same log file so the root dir is not full of logs
- move to python3 
- remove the escaping around the mongo stop cmd as the it is in "" so it is not needed
- update entrypoint with some prity version prints

should look something like this: but with no npm version print
image

### All Submissions:

* [x] Have you ensured there aren't other open [Pull Requests](../pulls) for the same update or change?
* [x] Have you created a new branch for your changes and PR from that branch and not from your master branch?

### CODE REVIEW

**Comment by gOOvER** [comment]
@parkervcp pls check this; it's needed for the ne bastionbot

File: bot/bastion/entrypoint.sh [hunk 1]
```diff
@@ -7,13 +24,19 @@ export INTERNAL_IP
...
 
 # Run the Server
+echo -e "${BLUE}-------------------------------------------------${NC}"
+echo -e "${YELLOW}BastionBot starting...${NC}"
+echo -e "${BLUE}-------------------------------------------------${NC}"
 eval ${MODIFIED_STARTUP}
 
 # stop mongo
-mongo --eval \"db.getSiblingDB('admin').shutdownServer()\
+mongo --eval "db.getSiblingDB('admin').shutdownServer()"
```
Comment (gOOvER) [inline bot/bastion/entrypoint.sh:42]: pls change this to

`mongod --shutdown`

**Review by gOOvER — CHANGES_REQUESTED** [review]

File: bot/bastion/entrypoint.sh (hunk 1 above)
Comment (QuintenQVD0) [reply bot/bastion/entrypoint.sh:42]: does it work if mongo is running in the background?

Comment (gOOvER) [reply bot/bastion/entrypoint.sh:42]: > does it work if mongo is running in the background?

i had an error, that "mongo" was not found; after changing it to mongod --shutdown it worked

But test it. Mongo 6 need mongod --shutdown

**Review by parkervcp — APPROVED** [review]

**Pull request merged**